cmd/pdc/main.go

```diff
@@ -107,6 +124,54 @@ func main() {
 		os.Exit(1)
 	}
 
+	sshKeyFileEnvVar, ok := os.LookupEnv("GCLOUD_SSH_KEY_FILE")
+	if ok && (sshConfig.KeyFile == "") {
+		sshConfig.KeyFile = sshKeyFileEnvVar
+	}
+
+	// Using the same logic as the `ssh.Config` definition, clamping to max of 3
+	sshLogLevelSelectEnvVar, envVarOk := os.LookupEnv("GCLOUD_SSH_LOG_LEVEL")
+	parsedSSHLogLevelSelectEnvVar, parseIntErr := strconv.Atoi(sshLogLevelSelectEnvVar)
+	if parseIntErr == nil && envVarOk && (sshConfig.LogLevel > 3 || sshConfig.LogLevel < 0) {
+		sshConfig.LogLevel = parsedSSHLogLevelSelectEnvVar
+	}
+
+	skipSSHValidationEnvVar, envVarOk := os.LookupEnv("GCLOUD_SSH_SKIP_SSH_VALIDATION")
+	parsedSSHValidationEnvVar, parseBoolErr := strconv.ParseBool(skipSSHValidationEnvVar)
+	if parseBoolErr == nil && envVarOk && !sshConfig.SkipSSHValidation {
+		sshConfig.SkipSSHValidation = parsedSSHValidationEnvVar
+	}
+
+	// We're actually going to append the values provided here to the already-parsed SSHFlags array
+	// As the argument can be provided more than once, this feels like the most compatible option
+	additionalSSHFlagsEnvVar, ok := os.LookupEnv("GCLOUD_SSH_ADDITIONAL_SSHFLAGS")
+	if ok {
+		sshConfig.SSHFlags = append(sshConfig.SSHFlags, strings.Split(additionalSSHFlagsEnvVar, " ")...)
+	}
+
+	sshForceKeyFileOverwriteEnvVar, envVarOk := os.LookupEnv("GCLOUD_SSH_FORCE_KEY_FILE_OVERWRITE")
+	parsedSSHForceKeyFileOverwriteEnvVar, parseBoolErr := strconv.ParseBool(sshForceKeyFileOverwriteEnvVar)
+	if parseBoolErr == nil && envVarOk && !sshConfig.ForceKeyFileOverwrite {
+		sshConfig.ForceKeyFileOverwrite = parsedSSHForceKeyFileOverwriteEnvVar
+	}
+
+	sshCertExpiryWindowEnvVar, envVarOk := os.LookupEnv("GCLOUD_SSH_CERT_EXPIRY_WINDOW")
+	parsedSSHCertExpiryWindowEnvVar, parseTimeErr := time.ParseDuration(sshCertExpiryWindowEnvVar)
+	if parseTimeErr == nil && envVarOk && (sshConfig.CertExpiryWindow == 0) {
+		sshConfig.CertExpiryWindow = parsedSSHCertExpiryWindowEnvVar
+	}
+
+	sshCheckCertExpiryPeriod, envVarOk := os.LookupEnv("GCLOUD_SSH_CERT_CHECK_EXPIRY_PERIOD")
+	parsedSSHCheckCertExpiryPeriod, parseTimeErr := time.ParseDuration(sshCheckCertExpiryPeriod)
+	if parseTimeErr == nil && envVarOk && (sshConfig.CertCheckCertExpiryPeriod == 0) {
```

Comment:
CertCheckCertExpiryPeriod has a default value of 1 minute, so this wont be hit.

Reply:
Addressed in 12810ff0a4715d90defd97a3fcda532bd4d2bceb